libs/SalesforceRestAPI/SalesforceRestAPI.xcodeproj/project.pbxproj

```diff
@@ -258,6 +292,16 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 		};
+		CEA8838F1C19135D008D871B /* Frameworks */ = {
+			isa = PBXFrameworksBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+				CEA883A61C191461008D871B /* libSalesforceSDKCore.a in Frameworks */,
```

Comment:
So we need to remove our own library dependencies in the ```Link Binary with Libraries``` step for the static targets, but ```Target Dependencies``` remain the same, correct? I'll make those changes.